pennylane/devices/qutrit_mixed/__init__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Submodule for performing mixed state simulations of qutrit-based quantum circuits.
+
+This submodule is internal and subject to change without a deprecation cycle. Use
+at your own discretion.
+
+TODO add auto-summary
```

Comment:
```suggestion
.. currentmodule:: pennylane.devices.qutrit_mixed
.. autosummary::
    :toctree: api

    create_initial_state
```

Reply:
If I recall the plan is to have a summary table below [here](https://xanaduai-pennylane--4861.com.readthedocs.build/en/4861/code/qml_devices.html#module-pennylane.devices.qubit). Note that this page isn't actually visible in the docs.

Reply:
Yep, but that requires the autosummary to be ready. I guess this suggestion doesn't need to be added until we actually include this module in the device docs, but it doesn't hurt to get it in.